pyaedt/modules/PostProcessor.py

```diff
@@ -1132,26 +1133,26 @@ def create_rectangular_plot(
             else:
                 families_input.append([families_dict[el]])
         self.oreportsetup.CreateReport(
-            plotname,
+            plot_name,
             modal_data,
             plot_type,
             setup_sweep_name,
             ctxt,
             families_input,
             ["X Component:=", primary_sweep_variable, "Y Component:=", expression],
         )
-        self.logger.info("Report %s correctly created.", plotname)
+        self.logger.info("Report %s correctly created.", plot_name)
         return True
 
     @pyaedt_function_handler()
     def get_solution_data_per_variation(
-        self, soltype="Far Fields", setup_sweep_name="", ctxt=None, sweeps=None, expression=""
+        self, solution_type="Far Fields", setup_sweep_name="", ctxt=None, sweeps=None, expression=""
```

Comment:
If we modify the name of the parameters to follow the snake_case convention, we might have to use a `deprecation warning` to allow the former scripts to keep working.